docs/.vuepress/config.js

```diff
@@ -41,6 +42,15 @@ if (getEnvDocsVersion()) {
 } else if (getEnvDocsFramework()) {
   versionPartialPath = '**/';
   frameworkPartialPath = `${getEnvDocsFramework()}${FRAMEWORK_SUFFIX}/`;
+
+} else if (isEnvDev()) {
+  // eslint-disable-next-line no-console
+  console.error(chalk.red(
+    'Stopping `docs:start` script execution. Any from `DOCS_FRAMEWORK` or `DOCS_FRAMEWORK` environment variables ' +
+    'should be defined for performance reason, ie. by using `DOCS_FRAMEWORK=javascript npm run docs:start:no-cache` ' +
+    'command.'
+  ));
+  process.exit(1);
```

Comment:
I'd rephrase (and reformat) that to something more along the lines of

```suggestion
  console.error(
    `${chalk.red(`\
Stopping the ${chalk.italic('docs:start')} script execution.

For performance reasons, the ${chalk.italic('DOCS_VERSION')} and/or ${chalk.italic('DOCS_FRAMEWORK')} \
environment variables need to be defined.

For example, try calling:
${chalk.italic.bold('DOCS_VERSION=next DOCS_FRAMEWORK=javascript npm run docs:start:no-cache')}.
`)}`);

  process.exit(1);
```

Reply:
Thank you, I've changed your suggestion just a little bit within: https://github.com/handsontable/handsontable/pull/9488/commits/7fec930d5e459482f10d6a1c8cc088a8cb238299.